src/modules/database/index.ts

```diff
@@ -69,6 +69,8 @@ export class Database {
    * faker.database.mongodbObjectId() // 'e175cac316a79afdd0ad3afb'
    */
   mongodbObjectId(): string {
-    return this.faker.datatype.hexadecimal({ length: 24, case: 'lower' });
+    return this.faker.datatype
+      .hexadecimal({ length: 24, case: 'lower' })
+      .replace('0x', '');
```

Comment:
You are getting creative with the removes?^^
```suggestion
      .hexadecimal({ length: 24, case: 'lower', prefix: '' });
```